project/profiles/capacity4more/modules/c4m/admin/c4m_admin/c4m_admin.module

```diff
@@ -24,3 +24,25 @@ function c4m_admin_menu() {
 
   return $items;
 }
+
+/**
+ * Returns site administrators ids.
+ *
+ * @return array
+ *   Ids of site admins
+ */
+function c4m_admin_get_admins_ids() {
+  // Get all site administrators.
+  $admin_role = user_role_load_by_name('administrator');
+  $results = db_select('users_roles', 'r')
+    ->fields('r', array('uid'))
+    ->condition('rid', $admin_role->rid, '=')
+    ->execute()
+    ->fetchAll();
+
+  foreach ($results as &$result) {
+    $result = $result->uid;
+  }
+
+  return empty($results) ? array() : $results;
```

Comment:
@anvmn 
when the `$results` are empty, does it means an empty array ?
if not then what happens on the `foreach` ?